src/TaskRunner.php

```diff
@@ -64,26 +65,38 @@ class TaskRunner
      */
     private $workingDir;
 
+    /**
+     * @var array
+     */
+    private $defaultCommandClasses = [
```

Comment:
Do we need to hardcode this ? Can't we get them autodetected by the classloader ?

Reply:
Yep this is a improvement, we can achieve that by moving our default command classes from `src/Commands` to `src/TaskRunner/Commands` as this will allow Robo to pick them up. We need to change their namespace too obviously.

If we do this we can remove the hard-coded classes and related code.

Reply:
I have something working locally, planning to polish it and add it here in less than 1 hour.